subprojects/docs/src/docs/release/notes.md

```diff
@@ -1,6 +1,8 @@
 The Gradle team is excited to announce Gradle @version@.
 
-This release features [1](), [2](), ... [n](), and more.
+This release adds [built-in dependency checksum and signature verification](#dependency-verification) to keep builds safe, provides a [shareable read-only dependency cache](#shared-dependency-cache) and contains [helpful documentation to avoid deprecations in your build](#deprecation-messages).
```

Comment:
How about "shareable read-only dependency cache **to accelerate ephemeral builds**"?

Reply:
removed